src/server/coprocessor/aggregate.rs

```diff
@@ -38,3 +39,23 @@ impl AggrFunc for Count {
         Ok(())
     }
 }
+
+type First = Option<Datum>;
+
+impl AggrFunc for First {
+    fn update(&mut self, mut args: Vec<Datum>) -> Result<()> {
+        if self.is_some() {
```

Comment:
What's mean by is_some()? Will it guarantee that the column value will be saved only once?